### ISSUE TITLE
Live TV Channel Switching Issues

### ISSUE THREAD

**Issue by mathieugouin** [xbmc/xbmc#17454]
### Describe the bug
Here is a clear and concise description of what the problem is:

I experience a strange Kodi live TV channel switching behavior where I loose the ability to futher switch channel and display program information.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

I would  expect that:
- Information would be displayed for all channels, even after switching to another channel.
- Up / Down arrow to continue working even after switching to another channel.

## Actual Behavior

- Channel switching with up/down arrow stops working
- Program information ('i' key) not showing information anymore

## Possible Fix

When going back to channel list and chosing the next channel from there, the information works and channel switching again works but for only one next channel.

### To Reproduce

Here are the steps to reproduce:
- In kodi, go to TV, Channels
- Select a channel to watch (press Enter)
- Wait for the channel to display full screen
- Press 'i' key to show information about the channel & description, confirm you see the channel number, channel icon, program description. (refer to 1st screen shot below)
- Press 'i' key to remove the information overlay window.
- Press Up or Down arrow to select another channel, press Enter to confirm.
- Wait for the next channel to display.  Confirm the new channel is playing.
- Press 'i' key to show information about the channel & description.  At this point, a 'not available' information window is displayed (issue nb. 1) (refer to 2nd screen shot below)
- Press 'i' key to remove the empty information overlay window.
- Press Up or Down arrow to select another channel.  At this point, nothing happens, as if the up/down keys are not working anymore (issue nb. 2).  But other keys seems to work, eg: x to close the channel.

### Debuglog

The debuglog can be found here: http://ix.io/2dLz

Dmesg: http://ix.io/2dLA

### Screenshots 
Here are some links or screenshots to help explain the problem:

- Working channel info: https://imgur.com/53TRjBt
- Not working channel info: https://imgur.com/8KbEhHX

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] Linux
 - [ ] OSX
 - [x] Raspberry-Pi
 - [ ] Windows
 - [ ] Windows UWP

- Operating system version/name: LibreELEC 9.2.0
- Kodi version: 18.5
- Tvheadend HTSP Client addon version 4.4.20.1
- Tvheadend server: Tvheadend 4.2.8-36

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by schornstephan** [comment]
My Kodi setup respectively my TVHeadend client plugin is affected by exactly the same problem. I also run Libreelec on a Raspberry Pi, version 3B+. The problem happens no matter whether the Tvheadend server runs on the same or another machine (Ubuntu 19.10 in my case).

I think Kodi "forgets" that it is handling a TV stream after switching channels via P+ button and falls back to the regular video mode. The controls Kodi's GUI is offering after switching channels resemble the controls Kodi offers when it is playing a video out of some folder.

**Comment by mathieugouin** [comment]
@schornstephan Thanks for your update on this issue!  What is the P+ button?  Do you mean page-up?   I forgot to say I'm using a wireless remote keyboard to do my Kodi navigation.

**Comment by mathieugouin** [comment]
I just found today that the issue is not only related to the up/down for switching channels.  It also happens when using the 'c' key to select another channel to watch.  I did not change the mapping, but I cannot find it on the official documentation.  It is referred as "Contextual menu" (https://kodi.wiki/view/Keyboard_controls).

To sum-up:
- press 'c'
- use arrows keys to highlight another channel
- press enter to select it
- confirm new channel is displayed
- press 'c' again, nothing happens
